src/gnn_reco/models/utils.py

```diff
@@ -0,0 +1,7 @@
+from torch_geometric.utils.homophily import homophily
+def calculate_xyzt_homophily(x,edge_index, batch):
```

Comment:
To an "outsider," at least, this is not a standard approach, and so a docstring detailing the intention behind this function would be useful.

Reply:
Thanks. Added docstring.